applications/jonny5/src/j5_channels.erl

```diff
@@ -328,6 +328,27 @@ per_minute_cost(AccountId) ->
                         call_cost(Channel) + Cost
                 end, 0, ets:select(?TAB, MatchSpec)).
 
+-spec real_per_minute_cost(ne_binary()) -> non_neg_integer().
+real_per_minute_cost(AccountId) ->
+    MatchSpec = [{#channel{account_id = AccountId
+                           ,account_billing = <<"per_minute">>
+                           ,_='_'
+                          }
+                  ,[]
+                  ,['$_']
+                 }
+                 ,{#channel{reseller_id = AccountId
+                            ,reseller_billing = <<"per_minute">>
+                            ,_='_'
+                           }
+                   ,[]
+                   ,['$_']
+                  }
+                ],
+    lists:foldl(fun(Channel, Cost) ->
```

Comment:
But `ets:foldl/3` traversed over entire table, `lists:foldl/3` work with allready filtered data.
Is `ets:foldl` more efficient then `ets:select` + `lists:foldl`  ?